config.yaml

```diff
@@ -0,0 +1,17 @@
+# This file contains default non-project-specific settings for 'stack', used
+# in all projects.  For more information about stack's configuration, see
+# http://docs.haskellstack.org/en/stable/yaml_configuration/
```

Comment:
Seems like this file has been added by mistake.

Reply:
Yep.